tests/test_app/TestApp/Stub/Stub.php

```diff
@@ -15,4 +15,9 @@ public function setProps($name)
     {
         $this->_setModelClass($name);
     }
+
+    public function getProp($name)
```

Comment:
Should we typehint string here too?

Reply:
This is a test case, not worth bothering.

Reply:
setProps($name) => setProps(?string $name)

Reply:
My logic is: If it is used wrong, then the expected results are also not what you want.
So probably worth also in those cases. But no biggie.